archai/nlp/nvidia_transformer_xl/nvidia_utils/exp_utils.py

```diff
@@ -238,8 +238,8 @@ def dataset_dir_name(dataset:str)->str:
         return 'wikitext-103'
     if dataset=='wt2':
         return 'wikitext-2'
-    if dataset=='olx':
-        return 'olx'
+    if 'olx' in dataset:
```

Comment:
I would say that it might be interesting to change this function to a dictionary in the near future as the amount of datasets might probably increase. Accessing dictionaries should be less expensive than checking for "ifs".